index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Survey Form microverse</title>
+</head>
+
+<body>
+    <div id="parent title">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+    </div>
+    <div id="form">
+        <form id="survey-form" action="">
+            <label id="name-label" for="name">
+                <h3>Name</h3>
+            </label>
+            <input id="name" type="text" placeholder="Enter your name">
+            <label for="email" id="email-label">
+                <h3>Email</h3>
+            </label>
+            <input type="email" name="" id="email" placeholder="Enter your Email">
```

Comment:
You shouldn't let an attribute (name="") undefined

Reply:
- [ ] Again, please make sure not to give orders, and always give a reason as to why a change is required.